Sources/TidepoolKit/TAPI.swift

```diff
@@ -39,43 +36,57 @@ public class TAPI {
     /// The URLSessionConfiguration used for all requests. The default is typically acceptable for most purposes. Any changes
     /// will only apply to subsequent requests.
     public var urlSessionConfiguration: URLSessionConfiguration {
```

Comment:
Why is this public when there is a `setURLSessionConfiguration` below? Is this redundancy needed?

Reply:
Good observation. Will try to resolve this redundancy.